seed.d/rules.json

```diff
@@ -2,29 +2,36 @@
     {
         "rule_id": "cloud_instance_ros_evaluation|CONSUMPTION_MODEL",
         "description":"This instance is idling so consider switching to a consumption model.",
-        "reason": "Based on the host metrics set, we have detected that this {system.instance_type} instance on {system.cloud_provider.upper()} is displaying a higher amount of idling cycles within a 24h time frame. It can be potentially explained by the non-business hours. This idling state may be caused by picking an inappropriate consumption model.",
-        "resolution": "Please verify that this instance and its workload is having an appropriate consumption model for this specific workload.",
-        "condition":"CPU Score + MEM Score + I/O Score (combined)  conditioned IDLE > 30% out of 24h,  are in between 3/15 and 14/15"
+        "reason": "Based on the host metrics set, we have detected that this {system.instance_type} instance on {system.cloud_provider.upper()} region {system.cloud_region} is displaying a higher amount of idling cycles within a 24h time frame. It can be potentially explained by the non-business hours. This idling state may be caused by picking an inappropriate consumption model.\nThe specific issues identified by ROS are:\n{system.summary}",
+        "resolution": "Please verify that this instance and its workload is having an appropriate consumption model for this specific workload.\nYou may also want to consider changing the instance type from {system.instance_type} to one of the following instance types, which fit the load better: {system.candidate_instance_types}",
+        "condition":"CPU and memory average usage is under 5% for 24 hours"
     },
     {
         "rule_id": "cloud_instance_ros_evaluation|INSTANCE_OVERSIZED",
         "description":"The instance is oversized and optimization is possible.",
-        "reason": "Based on the host metrics set, we have detected that this {system.instance_type} instance on {system.cloud_provider.upper()} is not consuming fully the provided resources. It is leading us to a judgement that it can be oversized. We apply the logic of combination of all 3 scores with the amount of time within 24h your system spends idling, excluding non business hours.",
-        "resolution": "Please pay attention to workloads running inside this RHEL instance, either utilizing your PCP tools or third party application specific tools to verify that the sizing of this ordered instance type from provider is optimized for your workload.",
-        "condition":"CPU Score + MEM Score + I/O Score (combined)  conditioned IDLE < 30% out of 24h,  are lower than 9/15"
+        "reason": "Based on the host metrics set, we have detected that this {system.instance_type} instance on {system.cloud_provider.upper()} region {system.cloud_region} is not consuming fully the provided resources. It is leading us to a judgement that it can be oversized.\nThe specific issues identified by ROS are:\n{system.summary}",
```

Comment:
@pgarciaq , one thing in case of `INSTANCE_OVERSIZED`, in sample output of advisor engine we are not getting `summary` information. Will it ok to skip this part from content.

Reply:
With [latest](https://gitlab.cee.redhat.com/insights-rules/insights-plugins/-/merge_requests/3810) merge request change on advisor rules, I can see messages were added for `INSTANCE_OVERSIZED` state as well. 

https://gitlab.cee.redhat.com/insights-rules/insights-plugins/-/blob/master/telemetry/rules/plugins/ros/ros_instance_evaluation.py#L80